### ISSUE TITLE
Feature Request: Disabling Extractors on Inputs

### ISSUE THREAD

**Issue by dubsout** [Graylog2/graylog2-server#2830]
I did some searching and wasn't able to find this request already, sorry in advance if I missed it.

It would be nice to be able to temporarily disable extractors while troubleshooting how you want them to work. This would remove the need to completely delete / remove the contents of the extractor if you need to set it aside for a little while. 

For instance I am working with an extractor and have some regex I don't necessarily want to take effect while testing some other extractors - disabling instead of deleting would be great in this case until you get it all worked out.

* Graylog Version: 2.1.1
* Elasticsearch Version: 2.x
* MongoDB Version: xx
* Operating System: Ubuntu 16.04
* Browser version: xx

**Comment by jalogisch** [comment]
if you move to pipelines you could just disconnect them ... but i can understand your feature request.

**Comment by dubsout** [comment]
Agreed and I am slowly moving some of my logic over to pipelines but am still using some of the more simple / static logic within extractors. Is this a best practice or do you see extractors going away completely?
